templates/master/01-master-kubelet/_base/units/kubelet-cleanup.service.yaml

```diff
@@ -0,0 +1,16 @@
+name: kubelet-cleanup.service
+enabled: true
+contents: |
+  [Unit]
+  Description=Kubernetes Kubelet After Reboot Cleanup
+  Before=kubelet.service
+
+  [Service]
+  Type=oneshot
+  ExecStartPre=/bin/rm -f /var/lib/kubelet/cpu_manager_state
+  ExecStartPre=/bin/rm -f /var/lib/kubelet/memory_manager_state
+
+  ExecStart=/bin/true
```

Comment:
this is now redundant

Reply:
Why? In case someone creates a schedulable master they still need this, right?